config/routes.rb

```diff
@@ -9,9 +7,14 @@
   delete 'sign_out', to: "sessions#destroy", as: 'sign_out'
 
 
+  resources :suburb
+  resources :message
```

Comment:
both resources and messages are api controllers and should be inside the api namespaces group